debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/history/JdbcSchemaHistory.java

```diff
@@ -184,7 +184,9 @@ public boolean storageExists() {
         boolean sExists = false;
         try {
             DatabaseMetaData dbMeta = conn.getMetaData();
-            ResultSet tableExists = dbMeta.getTables(null, null, config.getTableName(), null);
+            String databaseName = config.getDatabaseName();
+            ResultSet tableExists = dbMeta.getTables(databaseName != null ? databaseName : null,
```

Comment:
The `?:` oprator is superfluous, if `databaseName` is `null` then it can be used instead of constant.